### PR TITLE
Feat/react router initial

### PR DESCRIPTION
**Pull request by juliogontor**
Configuration of react router and basic example

### CODE REVIEW

File: src/components/login/container/Login.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+import { Link, Redirect} from 'react-router-dom';
+import FormErrors from '../../../helpers/FormErrors.js'
+
+class Login extends React.Component {
+
+    constructor(props) {
+        super(props);
+
+       // reset login status
+        //this.props.dispatch(userActions.logout());
```
Comment (tony1908) [inline src/components/login/container/Login.js]: Why are these comments here?

File: src/components/login/container/Login.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+            //<Redirect to="/home"/>
+        }
+    };
+
+    validateField(fieldName, value) {  
+        let fieldValidationErrors = this.state.formErrors;
+        let emailValid = this.state.emailValid;
+        let passwordValid = this.state.passwordValid;
+      
+          switch(fieldName) {
```
Comment (tony1908) [inline src/components/login/container/Login.js:45]: May be you can change this switch for something simpler.

File: src/components/login/container/Login.js [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+                </form>
+                <div className="panel panel-default">
+                    <FormErrors formErrors={this.state.formErrors} />
+                </div>
+            </div>
+        );
+    }
+}
+
+function mapStateToProps(state) {
```
Comment (tony1908) [inline src/components/login/container/Login.js]: Are you using redux for this app?

File: src/components/register/container/Register.js [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+            submitted: false
+        };
+
+        this.handleChange = this.handleChange.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this);
+    }
+
+    handleChange(e) {
+        const { name, value } = e.target;
+        //this.setState({ [name]: value });
```
Comment (tony1908) [inline src/components/register/container/Register.js]: do you need this comment?

File: src/components/register/container/Register.js [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+
+
+    handleSubmit(e){
+        e.preventDefault();
+        this.setState({ submitted: true });
+        const { userName, email, password } = this.state;
+        const { dispatch } = this.props;
+        if (userName && email && password) {
+            this.props.history.push('/home')
+            //<Redirect to="/home"/>
```
Comment (tony1908) [inline src/components/register/container/Register.js]: if you're not using this component, remove it.

File: src/components/register/container/Register.js [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+        this.setState({[name]: value}, 
+            () => { this.validateField(name, value) });
+    }
+
+
+    handleSubmit(e){
+        e.preventDefault();
+        this.setState({ submitted: true });
+        const { userName, email, password } = this.state;
+        const { dispatch } = this.props;
```
Comment (tony1908) [inline src/components/register/container/Register.js]: Are you using redux on this app.

**Review by tony1908 — CHANGES_REQUESTED** [review]

File: src/components/login/container/Login.js (hunk 2 above)
Comment (juliogontor) [reply src/components/login/container/Login.js:45]: i'm using in register with one field (userName) more

**Review by tony1908 — APPROVED** [review]

**Pull request merged**